Auth0/SafariWebAuth.swift

```diff
@@ -239,8 +239,8 @@ private func generateDefaultState() -> String? {
     var data = Data(count: 32)
     var tempData = data
 
-    let result = tempData.withUnsafeMutableBytes { (bytes: UnsafeMutablePointer<UInt8>) -> Int in
-        return Int(SecRandomCopyBytes(kSecRandomDefault, data.count, bytes))
+    let result = tempData.withUnsafeMutableBytes {
+        SecRandomCopyBytes(kSecRandomDefault, data.count, $0.baseAddress!)
```

Comment:
Why has this changed here? and why isn't this declared as a function parameter?

Reply:
Compiler warning fix, just happened to work on it at while I was in the code base.